### DISCUSSION TITLE
Doubts related to Octernship Projects

### DISCUSSION THREAD

**Discussion by dakshdeepHERE** [community/community#50127] [GitHub Education]
So I have few doubts related with the projects which I will mention below:
- We have two branches in each repo so in which repo we have to push the code in the main branch or the feedback branch?
- Is there a way to connect with my mentor as I can't see any mentor assigned to me. It would be a great help if I can contact the mentor.

**Comment by naimur-reza** [comment]
Regarding the first question, the answer would depend on the specific development workflow that your team is following. However, as a general guideline, you should push your code changes to the feedback branch, which is usually created for the purpose of code review and testing. Once your changes have been reviewed and tested, they can be merged into the main branch.

Regarding your second question, if you are participating in a program or course that has assigned mentors, you should reach out to the program/course administrator to inquire about the mentorship arrangement. They should be able to provide you with more information about the mentor assigned to you or assign a mentor if one has not yet been assigned. Additionally, some programs may have communication channels such as forums or email groups where you can ask for assistance from mentors or other participants.
